README.md

```diff
@@ -59,6 +59,46 @@ const config = {
 module.exports = { config };
 ```
 
+### API Configuration
+
+If you wish to use the Electron APIs then you will need to import the preload and main scripts. At the top of your preload:
+
+```ts
+import 'wdio-electron-service/preload';
+```
+
+And at the top of your main index file (app entry point):
+
+```ts
+import 'wdio-electron-service/main';
+```
+
+The APIs should now be available in tests. Currently available APIs: [`app`](https://www.electronjs.org/docs/latest/api/app), [`mainProcess`](https://www.electronjs.org/docs/latest/api/process), [`browserWindow`](https://www.electronjs.org/docs/latest/api/browser-window).
+
+```ts
+const appName = await browser.electronApp('getName');
+```
+
+### Custom Electron API
+
+You can also implement a custom API if you wish. To do this you will need to define a handler in your main process:
+
+```ts
+ipcMain.handle('wdio-electron', () => {
```

Comment:
Where does `ipcMain` come from?

Reply:
Good catch, could be confusing. It's just imported from the electron module, I'll make this more obvious.